packages/client/src/index.css

```
@@ -25,9 +25,37 @@
   line-height: 1.5;
 }
 
+/* ── Overlay layer scale (z-index) ────────────────────────────────
+   Single source of truth for stacking order. Portaled overlays
+   reference a layer UTILITY (z-popover, z-dialog, …) — never a raw
+   z-[NNNN], which carries no ordering intent and cannot be reasoned
+   about against the other overlays. In-flow, non-portaled decorations
+   (sticky headers, scrollbar shims) keep a small local z-* and are
+   out of scope. Ascending = paints later = on top.
+   See openspec spec overlay-layering. See change: add-overlay-layering-system. */
+@utility z-base { z-index: var(--z-base); }
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)